### DISCUSSION TITLE
Unexpected Indentation of row data after enabling Row Virtualization.

### DISCUSSION THREAD

**Discussion by abhiGunjan** [KevinVandy/material-react-table#939] [Q&A]
I observe automatically Indentation in MRT expandable table rows after enabling the prop enableRowVirtualization: true, I am trying to fix this but unable to do. 
Here is the codesandbox link, **(after turning enableRowVirtualization to false, It's rendering expected, But can't make it false as i have huge tree data)**  [Link](https://codesandbox.io/p/devbox/determined-haze-238fh7?file=%2Fsrc%2FTS.tsx%3A111%2C7-111%2C38) _How to fix this?_

1.  Expected Result
![image](https://github.com/KevinVandy/material-react-table/assets/79216975/a7ba957f-a161-4759-851c-5635519a9d1c)

2. Getting this 
![image](https://github.com/KevinVandy/material-react-table/assets/79216975/031e4b83-39d2-4dc6-bd51-5fb1671eb63b)

**Comment by KevinVandy** [comment]
You just need to make your expand column wider

**Reply by abhiGunjan** [reply]
Thank you for your replay , I tried widening the expand column ` displayColumnDefOptions: {
    'mrt-row-expand': {
      enableResizing: true, 
      size: 200
    },
  },` , but still it's not resolving the issue, I've update the sandbox link, Please help  me to fix this.

**Reply by abhiGunjan** [reply]
@KevinVandy  Sir, Can you please help me to fix this?

**Reply by KevinVandy** [reply]
What you tried should have fixed it

**Reply by abhiGunjan** [reply]
yes but unfortunately It did not solve.  without making enableRowVirtualization to false problem remains the same.

**Reply by KevinVandy** [reply]
Looking into this more, there is actually a slight problem with the implementation here. Working on some fixes. layoutMode: 'semantic' could be a solution for you if you can do that for now.

**Reply by abhiGunjan** [reply]
Hi, layoutMode: 'semantic' too did not work. :(

**Reply by KevinVandy** [reply]
Are you using virtualization?

**Reply by KevinVandy** [reply]
@abhiGunjan Try upgrading to v2.8.0

**Reply by abhiGunjan** [reply]
Ok sure , I'll try. Thank you

**Reply by abhiGunjan** [reply]
Hi, Is there any way to fix it, without upgrading ?
